index.html

```diff
@@ -18,8 +18,8 @@
     <section class="headline">
     <div class="flex-container">
         <h1>Hey There I'm Alex</h1>
-        <h2>I'am a Software Developer</h2>
-        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        <h2 class="pro">I'am a Software Developer</h2>
+        <p class="discription">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
             don’t hestiate to contact me.</p>
     </div>
    </section>
```

Comment:
## Optional Suggestions ♻♻
---

Your project - Html    | Microverse Best Practices
----------------------- | -----------------------------
![guidelines](https://user-images.githubusercontent.com/54015740/143720284-ee11ee91-9f11-42b9-8c54-9837a4a39e43.png)|![microverse](https://user-images.githubusercontent.com/54015740/143720278-b19f254a-29fe-4b1d-a8f0-9b691818aa6d.png)
- [ ] Remember to always read the best practices of Microverse for presentation of project guidelines, is very important and if previous reviewer noticed, your project would have been invalidated for not using the best practices, however, we will let it pass this time due having completed the required changes of previous reviewer, but make sure that in your next projects you apply the best practices rules of Microverse.
- [ ] Check Best practices #3 for indentation guidelines, too many blank spaces looks really bad for a project presentation, makes your project fail to be on the professionalism standards, so be careful with this part for next projects, and make sure to fix your project's appearance before merging , recruiters often check GitHub project's you've done, would be nice to implement this change👍 💯 

![tenor](https://user-images.githubusercontent.com/54015740/127129480-dfb12898-5429-4ff7-a67a-f4e3df0edc64.gif)

Reply:
😊 It looks better when you don't leave unnecessary blank spaces on your project as this piece of code signaled here above✔